apps/site/lib/site_web/router.ex

```diff
@@ -207,6 +206,7 @@ defmodule SiteWeb.Router do
     pipe_through([:secure, :browser])
 
     get("/alerts", AlertController, :show_by_routes)
+    get("/stops/:stop_id/alerts", AlertController, :show_by_stop)
```

Comment:
question: It makes sense to me that this is moved to be near the other routes using AlertController. Is there a reason `/stops` is still the first part of the path?

Reply:
Ah, I hadn't noticed this was moved to under `/api` prefix, thanks for pointing that out! Your description makes sense to me, thanks!